pkg/k8s/informer/informer.go

```diff
@@ -16,20 +16,44 @@ package informer
 
 import (
 	"fmt"
+	"net/http"
 	"time"
 
-	"k8s.io/apimachinery/pkg/runtime"
+	"github.com/cilium/cilium/pkg/logging"
+	"github.com/cilium/cilium/pkg/logging/logfields"
+
+	k8sRuntime "k8s.io/apimachinery/pkg/runtime"
+	utilRuntime "k8s.io/apimachinery/pkg/util/runtime"
 	"k8s.io/client-go/tools/cache"
 )
 
+var log = logging.DefaultLogger.WithField(logfields.LogSubsys, "k8s")
+
+func init() {
+	utilRuntime.PanicHandlers = append(
```

Comment:
Is this still required even if `ReallyCrash` is true? Who recovers?

https://github.com/kubernetes/apimachinery/blob/master/pkg/util/runtime/runtime.go#L33

Reply:
@tgraf `ReallyCrash` is true, since it runs in a go routine the panic is presented in logs but cilium does not crash.

>Who recovers?

the same entity that created the panic